smoke/runtime/runtime_test.go

```diff
@@ -175,3 +183,23 @@ func allTrue(bools []bool) bool {
 	}
 	return true
 }
+
+func curl(skip bool, url string) (string, error) {
```

Comment:
We dislike this function name, since it only performs a `GET` and `curl` is a more general purpose utility.  Can we rename the function to better describe the use case it supports?